addons/web_editor/static/src/js/wysiwyg/widgets/media.js

```diff
@@ -951,6 +951,7 @@ var ImageWidget = FileWidget.extend({
     _onSearchInput: function (ev) {
         this.libraryMedia = [];
         this._super(...arguments);
+        this.$('.o_we_search_select').removeClass('d-none');
```

Comment:
@younn-o I am moving this commit for 14.3 instead of 14.0 (not really stable, require rd-security approval, conflicts, etc)